src/coreclr/src/tools/Common/JitInterface/CorInfoBase.cs

```diff
@@ -2518,176 +2518,176 @@ static IntPtr GetUnmanagedCallbacks()
-            callbacks[147] = (delegate* <IntPtr, IntPtr*, IntPtr, CORINFO_METHOD_STRUCT_*, void>)&_setOverride;
-            callbacks[148] = (delegate* <IntPtr, IntPtr*, CORINFO_MODULE_STRUCT_*, CORINFO_MODULE_STRUCT_*, void>)&_addActiveDependency;
-            callbacks[149] = (delegate* <IntPtr, IntPtr*, CORINFO_METHOD_STRUCT_*, CORINFO_CLASS_STRUCT_*, CORINFO_METHOD_STRUCT_*, DelegateCtorArgs*, CORINFO_METHOD_STRUCT_*>)&_GetDelegateCtor;
-            callbacks[150] = (delegate* <IntPtr, IntPtr*, CORINFO_METHOD_STRUCT_*, void>)&_MethodCompileComplete;
-            callbacks[151] = (delegate* <IntPtr, IntPtr*, CORINFO_RESOLVED_TOKEN*, CORINFO_SIG_INFO*, CORINFO_GET_TAILCALL_HELPERS_FLAGS, CORINFO_TAILCALL_HELPERS*, byte>)&_getTailCallHelpers;
-            callbacks[152] = (delegate* <IntPtr, IntPtr*, CORINFO_RESOLVED_TOKEN*, byte, byte>)&_convertPInvokeCalliToCall;
-            callbacks[153] = (delegate* <IntPtr, IntPtr*, InstructionSet, byte, void>)&_notifyInstructionSetUsage;
-            callbacks[154] = (delegate* <IntPtr, IntPtr*, uint, uint, uint, uint, CorJitAllocMemFlag, void**, void**, void**, void>)&_allocMem;
-            callbacks[155] = (delegate* <IntPtr, IntPtr*, int, int, uint, void>)&_reserveUnwindInfo;
-            callbacks[156] = (delegate* <IntPtr, IntPtr*, byte*, byte*, uint, uint, uint, byte*, CorJitFuncKind, void>)&_allocUnwindInfo;
-            callbacks[157] = (delegate* <IntPtr, IntPtr*, UIntPtr, void*>)&_allocGCInfo;
-            callbacks[158] = (delegate* <IntPtr, IntPtr*, uint, void>)&_setEHcount;
-            callbacks[159] = (delegate* <IntPtr, IntPtr*, uint, CORINFO_EH_CLAUSE*, void>)&_setEHinfo;
-            callbacks[160] = (delegate* <IntPtr, IntPtr*, uint, byte*, IntPtr, int>)&_logMsg;
-            callbacks[161] = (delegate* <IntPtr, IntPtr*, byte*, int, byte*, int>)&_doAssert;
-            callbacks[162] = (delegate* <IntPtr, IntPtr*, CorJitResult, void>)&_reportFatalError;
-            callbacks[163] = (delegate* <IntPtr, IntPtr*, uint, BlockCounts**, HRESULT>)&_allocMethodBlockCounts;
-            callbacks[164] = (delegate* <IntPtr, IntPtr*, CORINFO_METHOD_STRUCT_*, uint*, BlockCounts**, uint*, HRESULT>)&_getMethodBlockCounts;
-            callbacks[165] = (delegate* <IntPtr, IntPtr*, uint, CORINFO_SIG_INFO*, CORINFO_METHOD_STRUCT_*, void>)&_recordCallSite;
-            callbacks[166] = (delegate* <IntPtr, IntPtr*, void*, void*, ushort, ushort, int, void>)&_recordRelocation;
-            callbacks[167] = (delegate* <IntPtr, IntPtr*, void*, ushort>)&_getRelocTypeHint;
-            callbacks[168] = (delegate* <IntPtr, IntPtr*, uint>)&_getExpectedTargetArchitecture;
-            callbacks[169] = (delegate* <IntPtr, IntPtr*, CORJIT_FLAGS*, uint, uint>)&_getJitFlags;
+            callbacks[0] = (delegate* unmanaged<IntPtr, IntPtr*, CORINFO_METHOD_STRUCT_*, uint>)&_getMethodAttribs;
```

Comment:
Nit: `unmanaged` should be also added here: https://github.com/dotnet/runtime/blob/master/src/coreclr/src/tools/Common/JitInterface/ThunkGenerator/Program.cs#L336